woocommerce/woocommerce-paypal-payments
# Create new connection tab

**Issue**: #801

---

### Description

The PR will create a new "Connection" tab with some settings should be migrated into it.

### Steps to Test




1. Go to Settings -> Connection Tab.
2. Check if appropriate fields are removed from other tabs.

---

Closes #801 .
